web/src/main/webapp/my-app/layout/partials/toast-widget-removal.html

```diff
@@ -0,0 +1,41 @@
+<!--
+
+    Licensed to Apereo under one or more contributor license
+    agreements. See the NOTICE file distributed with this work
+    for additional information regarding copyright ownership.
+    Apereo licenses this file to you under the Apache License,
+    Version 2.0 (the "License"); you may not use this file
+    except in compliance with the License.  You may obtain a
+    copy of the License at the following location:
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<md-toast class="toast__widget-removal">
+  <div class="md-toast-content" layout="column">
+    <div class="toast__message" layout="row" layout-align="space-between center">
+      <div class="toast__text" layout="column" layout-align="center start" flex="80">
+        <span class="toast__source" ng-style="{ color: accentColorRgb }">Confirm change</span>
+        <span>Removed {{ removedTitle }}</span>
+      </div>
+      <span class="toast__icon-container">
+        <widget-icon></widget-icon>
+      </span>
+    </div>
+    <div class="toast__actions" layout="row" layout-align="start center">
+      <md-button class="md-accent" ng-click="undoRemoveWidget($event)" aria-label="undo remove">
+        Undo
+      </md-button>
+      <md-button class="md-accent" ng-click="confirmRemoveWidget()" aria-label="confirm remove">
+        OK
```

Comment:
I'm not sure I like that the user has to confirm the change. I love that the user has opportunity to un-do. Would there be a way to do this that defaults to honoring the user's apparent intent once expressed yet nonetheless gives the user an easy way to un-do it in case they were misunderstood?

That is, I'm not sure a user should have to click twice to remove an item so much as I'm sure that once a user does remove an item they should be one click away from un-doing this.